packages/virtual-keyboard/src/virtual-keyboard/unique-keyboard-tray.tsx

```diff
@@ -3,13 +3,24 @@ import { createRoot, Root } from "react-dom/client";
 import { OnClick } from "./keyboard";
 import { KeyboardTray } from "./keyboard-tray";
 
-const virtualKeyboardState = {
-    count: 0,
-    keyboardDomNode: null as HTMLElement | null,
-    keyboardReactRoot: null as Root | null,
-    callbacks: [] as OnClick[],
+let virtualKeyboardState: {
+    count: number;
+    keyboardDomNode: HTMLElement | null;
+    keyboardReactRoot: Root | null;
+    callbacks: OnClick[];
 };
```

Comment:
This would look better defined as a type. They you can do


`const virtualKeyboardState: VirtualKeyboardState = (window as any).virtualKeyboardState || { ... }`

Reply:
done, though now I set window.virtualKeyboardState every time since I don't have an if statement